react-props-and-expressions/my-app/src/Btn-component.js

```diff
@@ -0,0 +1,7 @@
+export default function MyButton(text, style) {
```

Comment:
Ah, this won't be recognized as a React component because it takes more than one parameter. It can only take one, an object of props. This is why you had to call it strangely from App.